### ISSUE TITLE
Installed webextension on machine 1, was sync'ed, but no icon on machine B

### ISSUE THREAD

**Issue by old-o** [sarahavilov/secure-login#17]
First of all many thanks for the Secure Login extension which I have been using for several years!

Today I tried the new Secure Login webextension on computer A. 
I use sync to automatically have the same extensions (and passwords) on all my computers.

Later on computer B, the Secure Login webextension was correctly listed among my extensions (due to the automatic syncing). But its toolbar icon was not to be found anywhere, not on any of the toolbars and not in the area for unused available icons in the Customize dialog.

I did a remove/reinstall of the extension on computer B, but even that did not give me the icon.

For now, I went back to using the XUL based extension.

**Comment by sarahavilov** [comment]
To have Secure Login (WebExtension), you will need Firefox equal or greater than 52. Are you sure your Firefox is up-to-date?

btw, you can have WebExtension in one machine and XUL in the other one

**Comment by old-o** [comment]
Yes, that was probably the reason. Machine 1 was updated to version 52, while machine 2 was still at version 50 (on Debian where the auto-update is disabled by default). Now I just upgraded machine 2 and received the message about Secure Login being deprecated, and then I installed the webextension successfully on machine 2. So it was probably just the older FF version that caused the problem.

Thanks for spotting this and responding so quickly! :)
